desciption: use #pragma message to avoid warning turned into an error
author: Michael Gilbert <mgilbert@debian.org>

--- a/dlls/rpcrt4/cproxy.c
+++ b/dlls/rpcrt4/cproxy.c
@@ -172,7 +172,7 @@ static inline void init_thunk( struct th
 
 #else  /* __i386__ */
 
-#warning You must implement stubless proxies for your CPU
+#pragma message "warning You must implement stubless proxies for your CPU"
 
 struct thunk
 {
--- a/dlls/rpcrt4/cstub.c
+++ b/dlls/rpcrt4/cstub.c
@@ -176,7 +176,7 @@ static const BYTE opcodes[16] = { 0x48,
                                   0xff, 0xa0, 0, 0, 0, 0, 0x48, 0x8d, 0x36 };
 #else
 
-#warning You must implement delegated proxies/stubs for your CPU
+#pragma message "warning You must implement delegated proxies/stubs for your CPU"
 typedef struct
 {
     DWORD offset;
--- a/dlls/rpcrt4/ndr_stubless.c
+++ b/dlls/rpcrt4/ndr_stubless.c
@@ -1132,7 +1132,7 @@ __ASM_GLOBAL_FUNC( call_server_func,
                    "mov SP, r5\n\t"
                    "pop {r4, r5, PC}" )
 #else
-#warning call_server_func not implemented for your architecture
+#pragma message "call_server_func not implemented for your architecture"
 LONG_PTR __cdecl call_server_func(SERVER_ROUTINE func, unsigned char * args, unsigned short stack_size)
 {
     FIXME("Not implemented for your architecture\n");
